packages/core/admin/ee/server/controllers/admin.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+const { env } = require('@strapi/utils');
+const { getService } = require('../../../server/utils');
+
+module.exports = {
+  async licenseLimitInformation() {
+    const permittedSeats = strapi.ee.licenseInfo.seats;
+    if (!permittedSeats) {
+      return;
+    }
+
+    let shouldNotify = false;
+    let licenseLimitStatus = null;
+    let enforcementUserCount;
+
+    const currentActiveUserCount = await getService('user').getCurrentActiveUserCount();
+
+    const data = await getService('user').getDisabledUserList();
+
+    if (data.value) {
+      const eeDisabledUsers = JSON.parse(data.value);
+      enforcementUserCount = currentActiveUserCount + eeDisabledUsers.length;
+    } else {
+      enforcementUserCount = currentActiveUserCount;
+    }
+
+    if (enforcementUserCount > permittedSeats) {
+      shouldNotify = true;
+      licenseLimitStatus = 'OVER_LIMIT';
+    }
+
+    if (enforcementUserCount === permittedSeats) {
+      shouldNotify = true;
+      licenseLimitStatus = 'AT_LIMIT';
+    }
+
+    return {
+      data: {
+        enforcementUserCount,
+        currentActiveUserCount,
+        permittedSeats,
+        shouldNotify,
+        shouldStopCreate: currentActiveUserCount >= permittedSeats,
+        licenseLimitStatus,
+        isHostedOnStrapiCloud: env('STRAPI_HOSTING', null) === 'strapi.cloud',
```

Comment:
more of a product question! what's the difference if is hosted on strapi cloud? I am seeing in the tests that the billing url is different.

Is there any other difference?

Reply:
There will be a different url that the user is transferred to if they want to add more seats to their license. The rest of the process is the same